src/rpm/builder.rs

```diff
@@ -368,6 +370,11 @@ impl PackageBuilder {
         Ok(())
     }
 
+    pub fn pre_trans_script(mut self, content: impl Into<String>) -> Self {
+        self.pre_trans_script = Some(content.into());
+        self
+    }
```

Comment:
Would you mind adding docstrings for all 6 methods here?

Reply:
There's also some relevant information to know if you are using this feature.

This is a copy and paste, roughly, of what is already here for the other tags.  Unfortunately I'm pretty sure that what is already here is wrong or at least incomplete, including for the existing scriptlet tags.  I haven't had much time to look into it much yet but here's what I noticed:

For each of the scriptlet types there are actually 3 tags, each, there's the scriptlet itself, and also a set of flags and the command / arguments needed to execute it: https://github.com/rpm-software-management/rpm/blob/master/docs/manual/tags.md?plain=1#L189-L251

We're not currently doing anything with flags nor arguments.  I have no idea how important that is (as I said, haven't gotten around to it, and also I've never used these tags personally), but it is definitely incomplete.  There also appears to be `preuntrans` and `postuntrans` tags, which could probably be added in this PR.

IF you are up for it, it would be great if you could take a look into this and fix up the implementation a bit.  However, I don't want to pile on a bunch of extra work for you, and I won't require it to merge this PR or anything.  Any existing broken-ness is really a separate issue.

Reply:
@dralley Sure no problem, seems like it could be useful down the road. 

As an aside, do you happen to know why the scriptlet naming is different here, https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax compared to here https://rpm-software-management.github.io/rpm/manual/tags.html?

Just want to double check my doc-/comments are accurate. I'm guessing rpm.org is probably the source of truth?